### DISCUSSION TITLE
Odd Filename Ordering

### DISCUSSION THREAD

**Discussion by SimonPlata** [jarun/nnn#1951] [Show and tell]
When I first used nnn, I was surprised by the alphanumeric sorting order of file names. I wanted to achieve the order I was accustomed to seeing. After some research in the source code, I discovered that I could obtain it by changing LC_COLLATE to 'C'.

~$ export LC_COLLATE=C

Default filename sorting
![image](https://github.com/user-attachments/assets/13371bd7-fa0e-4a33-8e3a-8e0fd9fa6269)

Filename sorting with LC_COLLATE=C
![image](https://github.com/user-attachments/assets/884d9d42-049b-4744-a078-ab5948a757f5)

**Comment by jarun** [comment]
Thanks for sharing!
